lib/management.js

```diff
@@ -3,12 +3,44 @@ const forge = require('node-forge');
 const arsenal = require('arsenal');
 const config = require('../conf/Config');
 const AWS = require('aws-sdk');
-
-const MetadataFileClient = arsenal.storage.metadata.MetadataFileClient;
-const mdClient = new MetadataFileClient({
-    host: config.queuePopulator.dmd.host,
-    port: config.queuePopulator.dmd.port,
-});
+const Logger = require('werelogs').Logger;
+
+const logger = new Logger('mongoclient');
+
```

Comment:
It could be useful to add a configuation to werelogs so we can manage the log level from the config.json, e.g.:

```js
werelogs.configure({ 
    level: config.log.logLevel,
    dump: config.log.dumpLevel,
});
```